scripts/infra-mcp/utils/security.py

```
@@ -0,0 +1,77 @@
+Security utilities for validating and protecting against various attack vectors.
+"""
+
+import ipaddress
+import logging
+import socket
+from urllib.parse import urlparse
+
+logger = logging.getLogger(__name__)
+
+
+def validate_url_for_ssrf(url: str) -> bool:  # noqa: PLR0911
```

Ruff (0.13.1)

Unused `noqa` directive (non-enabled: `PLR0911`)

Remove unused `noqa` directive

(RUF100)